automated/linux/ssuite/run-bench.sh

```diff
@@ -173,18 +173,18 @@ run_test() {
 	rm -f ${HOME_DIR}/.S-config.sh
 
 	cd "$S_PATH"/run_multiple_benchmarks/ || exit 1
-	./run_main_benchmarks.sh "$1" "" "" "" "${ONLY_READS}" "" 2 "${OUTPUT}" 2>&1 |\
+	./run_main_benchmarks.sh "$1" "" "" "" "${ONLY_READS}" "" 5 "${OUTPUT}" 2>&1 |\
```

Comment:
Would it be possible to add number of repetitions as a variable that can be changed from the ssuite-bench.yaml and add a default value "5" ?

Reply:
Absolutely! Just let me first understand what Milosz is trying to explain to me on the multiple-value issue.